cmd/queue/main.go

```diff
@@ -366,17 +410,16 @@ func main() {
 	// Create queue handler chain
 	// Note: innermost handlers are specified first, ie. the last handler in the chain will be executed first
 	var composedHandler http.Handler = httpProxy
-	if metricsSupported {
-		composedHandler = pushRequestMetricHandler(httpProxy, appRequestCountM, appResponseTimeInMsecM, env)
-	}
```

Comment:
Is this removed intentionally?

Reply:
no - i wonder if there's something strange about the way i merged master into this branch?  Not sure how else this could be removed.